store/persistent/storethehash/storethehash.go

```diff
@@ -212,3 +213,10 @@ func (s *sthStorage) removeEntry(k []byte, entry store.IndexEntry, stored []stor
 	}
 	return false, nil
 }
+
+// Close stops all storage-related routines, and flushes
+// pending data
+// NOTE: Add it to storage interface?
+func (s *sthStorage) Close() error {
+	return s.store.Close()
```

Comment:
It probably makes sense to expose this through the Storage interface to make sure data is flushed at shutdown.  Would there be any utility in exposing a `Flush` method so that a library user could call that after finishing some bulk ingestion?

Reply:
I exposed `Close` in the persistent storage interface. I haven't removed `Flush` because it is being used by unit tests to avoid having to wait for the next sync tick. I checked pogreb's API and it exposes both, Close and Flush, but let me know if you think it'd be better to remove Flush from the interface.